<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraWindow</class>
 <widget class="QMainWindow" name="CameraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>793</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Experiment.qrc">
    <normaloff>:/openicon/icons/movie.png</normaloff>:/openicon/icons/movie.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionCoolCCD"/>
   <addaction name="actionAcquire"/>
   <addaction name="actionLive"/>
   <addaction name="actionReset"/>
   <addaction name="actionClearQueue"/>
   <addaction name="actionCalibrate_ROI"/>
  </widget>
  <widget class="QDockWidget" name="settingsDock">
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="temperatureDock">
   <property name="windowTitle">
    <string>Temperature</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/kde/icons/filesave.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAcquire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Z:/Lab/Andor Project/imaging_software_g/bitmaps/go.png</normaloff>Z:/Lab/Andor Project/imaging_software_g/bitmaps/go.png</iconset>
   </property>
   <property name="text">
    <string>Acquire</string>
   </property>
   <property name="toolTip">
    <string>Acquire</string>
   </property>
  </action>
  <action name="actionCoolCCD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/go-down-7.png</normaloff>:/openicon/icons/go-down-7.png</iconset>
   </property>
   <property name="text">
    <string>CoolCCD</string>
   </property>
   <property name="toolTip">
    <string>Cool CCD</string>
   </property>
  </action>
  <action name="actionLive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/petersIcons/icons/ChipScope.png</normaloff>:/petersIcons/icons/ChipScope.png</iconset>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
   <property name="toolTip">
    <string>Live Mode</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset>
     <normalon>:/openicon/icons/view-refresh-7.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Reset View</string>
   </property>
  </action>
  <action name="actionClearQueue">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/kde/icons/edit-clear.png</normaloff>:/kde/icons/edit-clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Queue</string>
   </property>
  </action>
  <action name="actionCalibrate_ROI">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/office-chart-bar.png</normaloff>:/openicon/icons/office-chart-bar.png</iconset>
   </property>
   <property name="text">
    <string>Calibrate ROI</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
